src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -25,6 +26,12 @@ Screenshots may contain <PlatformLink to="/data-management/sensitive-data/">PII<
 
 <PlatformContent includePath="enriching-events/attach-screenshots" />
 
+<PlatformSection supported={["react-native"]}>
+
+Screenshots are transfered from native SDKs to React Native over the Bridge. This can be a performance bottleneck on lower end devices. This happens only when capturing error event with the screenshots enabled. React Native New Architecture will improve the performance.
```

Comment:
Can we rearrange it like so?

Also, it doesn't look like bridge needs to be capitalized based on these docs [here](https://reactnative.dev/docs/native-modules-ios#reserved-method-names). But we can also get around this by using 
```suggestion
Screenshots are transferred from native SDKs to React Native over the `Bridge`, and this happens only when an error event is captured with screenshots enabled. This can negatively impact performance on lower end devices, however, React Native New Architecture will improve performance.
```